nuxeo-services/nuxeo-platform-web-common/src/main/java/org/nuxeo/ecm/platform/web/common/requestcontroller/filter/NuxeoRequestControllerFilter.java

```diff
@@ -134,6 +134,8 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
             boolean isCached = config.isCached();
             if (isCached) {
                 addCacheHeader(httpResponse, config.isPrivate(), config.getCacheTime());
+            } else if (config.isPrivate()) {
+                httpResponse.setHeader("Cache-Control", "private, no-cache, no-store, must-revalidate");
```

Comment:
Could you do the same changes as in master and move this code to `addCacheHeader`?

Reply:
Not that DRY but tried to keep current public API just in case

Reply:
👍